pkg/util/net/tls.go

```diff
@@ -39,6 +39,15 @@ func GetTLSConfig(certFile, keyFile, caFile string) (*tls.Config, error) {
 		Certificates: certs,
 		ClientCAs:    cas,
 		RootCAs:      cas,
+		CipherSuites: Safe,
 	}
```

Comment:
You might want to add:
* `PreferServerCipherSuites: true` to make sure the ordering of the server is honored
* `MinVersion:   tls.VersionTLS12` to set the minimum TLS version to 1.2